builds/misc/packages-release.yaml

```diff
@@ -202,6 +200,159 @@ stages:
+            artifactName: iotedged-ubuntu18.04-amd64
+            identityServiceArtifactName: packages_ubuntu-18.04_amd64
+            identityServicePackageFilter: aziot-identity-service_*_amd64.deb
+          Ubuntu1804-arm32v7:
+            os: ubuntu18.04
+            artifactName: iotedged-ubuntu18.04-arm32v7
+            identityServiceArtifactName: packages_ubuntu-18.04_arm32v7
+            identityServicePackageFilter: aziot-identity-service_*_armhf.deb
+          Ubuntu1804-aarch64:
+            os: ubuntu18.04
+            artifactName: iotedged-ubuntu18.04-aarch64
+            identityServiceArtifactName: packages_ubuntu-18.04_aarch64
+            identityServicePackageFilter: aziot-identity-service_*_arm64.deb
+
+          Ubuntu2004-amd64:
+            os: ubuntu20.04
+            artifactName: iotedged-ubuntu20.04-amd64
+            identityServiceArtifactName: packages_ubuntu-20.04_amd64
+            identityServicePackageFilter: aziot-identity-service_*_amd64.deb
+          Ubuntu2004-arm32v7:
+            os: ubuntu20.04
+            artifactName: iotedged-ubuntu20.04-arm32v7
+            identityServiceArtifactName: packages_ubuntu-20.04_arm32v7
+            identityServicePackageFilter: aziot-identity-service_*_armhf.deb
+          Ubuntu2004-aarch64:
+            os: ubuntu20.04
+            artifactName: iotedged-ubuntu20.04-aarch64
+            identityServiceArtifactName: packages_ubuntu-20.04_aarch64
+            identityServicePackageFilter: aziot-identity-service_*_arm64.deb
+      steps:
+        - task: AzureKeyVault@1
+          displayName: Get secrets
+          inputs:
+            azureSubscription: $(az.subscription)
+            keyVaultName: $(kv.name)
+            secretsFilter: >-
+              GitHubAccessToken
+        - task: DownloadPipelineArtifact@2
+          displayName: Download Pipeline Build Packages
+          inputs:
+            source: 'current'
+            path: $(System.ArtifactsDirectory)
+            patterns: |
+                $(artifactName)/*.deb
+                $(artifactName)/*.rpm
+        - task: PowerShell@2
+          displayName: 'Download aziot-identity-service'
+          inputs:
+            filePath: $(Build.SourcesDirectory)/scripts/local/test/DownloadIdentityService.ps1
+            workingDirectory: $(Build.SourcesDirectory)
+          env:
+            GITHUB_TOKEN: $(GitHubAccessToken)
+            ARTIFACT_NAME: $(identityServiceArtifactName)
+            PACKAGE_FILTER: $(identityServicePackageFilter)
+            DOWNLOAD_PATH: $(System.ArtifactsDirectory)/$(artifactName)
+            IDENTITY_SERVICE_COMMIT: $(aziotis.commit)
+        - bash: |
+            BASE_VERSION=`cat $BUILD_SOURCESDIRECTORY/edgelet/version.txt`
+            $(Build.SourcesDirectory)/scripts/linux/publishReleasePackages.sh -p $(os) -w $(System.ArtifactsDirectory) -d $(System.ArtifactsDirectory)/$(artifactName) -v "$BASE_VERSION" -s "github.com" -b $(Build.SourceBranch)
```

Comment:
Please correct me if I'm wrong. I think you still needs the `-g $GITHUB_PAT` here

Reply:
I dont think so, since its now defined as an environment variable. 

Example here : https://github.com/Azure/iotedge/blob/release/1.2/builds/e2e/isa-95-smoke-test.yaml#:~:text=steps%3A,name%3A%20lock_test_agent

Reply:
I ran a test run and found no issues : https://dev.azure.com/msazure/One/_build/results?buildId=50775349&view=results